app/src/main/java/nerd/tuxmobil/fahrplan/congress/models/DateInfo.java

```diff
@@ -1,18 +1,20 @@
 package nerd.tuxmobil.fahrplan.congress.models;
 
 
+import info.metadude.android.eventfahrplan.commons.Clock;
+
 public class DateInfo {
 
     public int dayIdx;
 
-    public String date;
+    public Clock date;
```

Comment:
Contract issues done as well as migration to Kotlin (LOCs reduced by 70%). 
I did not change the naming though. Because it has already been "date" before and the variable represents a date (variables of type String are not named "string" after all ;)).

Reply:
Thanks.